homeassistant/components/sabnzbd/sab.py

```diff
@@ -0,0 +1,27 @@
+"""Support for the Sabnzbd service."""
+from pysabnzbd import SabnzbdApi, SabnzbdApiException
+
+from homeassistant.const import CONF_API_KEY, CONF_PATH, CONF_URL
+from homeassistant.core import _LOGGER, HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+
+async def get_client(hass: HomeAssistant, data):
```

Comment:
Please add type annotations for the whole signature if adding any type annotations for a function.